01/app.js

```diff
@@ -1 +1,17 @@
-console.log('DOM');
\ No newline at end of file
+const commentsItemNewestList = document.querySelectorAll('.comments__item.comments__item--newest'); // uzyłam querySelectorAll by sprawdzić czy nie ma więcej takich elementów z podwójną klasą
+if (commentsItemNewestList) { //chyba nie potrzebne tutaj?
```

Comment:
Jak wyszukujemy wiele elementów to jest niepotrzebne bo pusta tablica to też prawda.
Dlatego ten warunek nic nie nie daje. Jak już to `commentsItemNewestList.length > 0`

Reply:
Zmienione